### PR TITLE
feat(curriculum): add two tests to challenge

### PR DESCRIPTION
**Pull request by gikf**
- Adds two tests.
- Removes not needed link.
- Slightly changes wording in tests description.
- Partially related to #40896.
- Tested on local fork.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/abundant-deficient-and-perfect-number-classifications.md [hunk 1]
```diff
@@ -32,32 +32,38 @@ Implement a function that calculates how many of the integers from `1` to `20,00
...
+assert.deepEqual(getDPA(10000), [7508, 4, 2488]);
+```
 
-## --after-user-code--
+`getDPA(20000)` should return `[15043, 4, 4953]`.
 
 ```js
-const solution = [15043, 4, 4953];
+assert.deepEqual(getDPA(20000), [15043, 4, 4953]);
 ```
```
Comment (nhcarrigan) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/abundant-deficient-and-perfect-number-classifications.md:63]: This has got to be one of the most confusing diff I have seen lol. Took me a minute to get it.

**Review by naomi-lgbt — APPROVED** [review]
This looks good to me.

**Review by Sembauke — APPROVED** [review]

**Comment by litoopcmbjp340** [comment]
Convesation ques are not ligitmized

DUSTTOODIAMONDS ™®

**Comment by litoopcmbjp340** [comment]
Try mergering que nano AA5549592024 no bank account attached.

DUSTTOODIAMONDS ™®

**Pull request merged**